### PR TITLE
Kf 942 gh188 feat auto build and scan

### PR DESCRIPTION
**Pull request by i-chvets**
See https://github.com/canonical/notebook-operators/issues/188 for more details.

### CODE REVIEW

File: scripts/send-scan.py [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+        for tag in record_rule["properties"]["tags"]:
+            if tag in ["CRITICAL", "HIGH", "MEDIUM", "LOW"]:
+                record_severity = tag
+        record_list.append(
+            {
+                "name": record_name,
+                "severity": record_severity,
+                "rule": record_rule,
+                "result": record_result,
+            }
```
Comment (beliaev-maksim) [inline scripts/send-scan.py:86]: need to update the dict to match JSON parser

Comment (i-chvets) [reply scripts/send-scan.py:86]: Will do.

Comment (i-chvets) [reply scripts/send-scan.py]: Done.

File: scripts/tests/test-empty-report.json [hunk 2]
```diff
@@ -0,0 +1,70 @@
+{
```
Comment (ca-scribner) [inline scripts/tests/test-empty-report.json:1]: Are these test files for some automated testing that wasn't committed?  Or are they meant for manual testing?  They feel like something a reader wont know what to do with

File: scripts/send-scan.py [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+        for record in records:
+            requests.post(jira_url, json=record)
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--report-path")
+    parser.add_argument("--jira-url")
+    args = parser.parse_args()
+    main(args.report_path, args.jira_url)
```
Comment (ca-scribner) [inline scripts/send-scan.py:128]: This isn't too complicated, but maybe still some explanation is needed.  Can we add a readme.md that explains how this is to be used?  I'm thinking something like [here](https://github.com/canonical/kubeflow-ci/tree/kf-942-gh188-feat-auto-build-and-scan/scripts/release-charms) (not necessarily as detailed, but a basic usage guideline and why someone should care)

**Pull request merged**